src/main/java/org/apache/commons/imaging/formats/jpeg/iptc/IptcParser.java

```diff
@@ -248,6 +258,43 @@ public PhotoshopApp13Data parsePhotoshopSegment(final byte[] bytes,
         return elements;
     }
 
+    private String getEncodingCharsetName(byte[] codedCharacterSet){
+        String codedCharacterSetString = new String(codedCharacterSet);
+        //byte[][] = getListOfEncoding
+        try {
+            if (Charset.isSupported(codedCharacterSetString)) {
+                return codedCharacterSetString;
+            }
+        }catch (IllegalCharsetNameException e){
+
+        }catch (IllegalArgumentException e){
+
+        }
+        //check if encoding is a escape sequence
+        //normalize encoding byte sequence
+        byte[] codedCharacterSetNormalized = new byte[codedCharacterSet.length];
+        int j=0;
+        for(int i=0; i< codedCharacterSet.length; i++){
+            if(codedCharacterSet[i] != ' ') {
+                codedCharacterSetNormalized[j++] = codedCharacterSet[i];
+            }
+        }
+        for(CharsetEscapeSequence escapeSeq : CharsetEscapeSequence.getSupportedEscapeSeqList()){
```

Comment:
@britter removing this logic for now. We would add this later if we need to add multiple escape sequences.